pkg/manifest.go

```diff
@@ -17,6 +17,7 @@ type PupManifest struct {
 	PermissionGroups []PermissionGroup `json:"permissionGroups"`
 	Dependencies     []Dependency      `json:"dependencies"`
 	hydrated         bool
+	containerIP      string
```

Comment:
The PupState for installed pups has the IP of the container

Reply:
Yup, this was temporary until we landed some of your changes.